common/openpype_common/distribution/file_handler.py

```diff
@@ -111,13 +111,15 @@ def download_url(
 
         # download the file
         try:
-            print('Downloading ' + url + ' to ' + fpath)
+            print(f"Downloading {url} to {fpath}")
             RemoteFileHandler._urlretrieve(url, fpath, headers=headers)
         except (urllib.error.URLError, IOError) as e:
-            if url[:5] == 'https':
-                url = url.replace('https:', 'http:')
-                print('Failed download. Trying https -> http instead.'
-                      ' Downloading ' + url + ' to ' + fpath)
+            if url[:5] == "https":
+                url = url.replace("https:", "http:")
+                print((
+                    "Failed download. Trying https -> http instead."
+                    f" Downloading {url} to {fpath}"
+                ))
                 RemoteFileHandler._urlretrieve(url, fpath,
```

Comment:
Shouldn't we catch a possible Exception on the second try with `https` ?

Reply:
No, it's catched where the `download` is called (it would not know if and how it crashed).